### ISSUE TITLE
Reset input / Undo

### ISSUE THREAD

**Issue by Azathoth88** [dopplertask/dopplertask#30]
I have just entered data on a test basis and overwritten my server data (MySQL action). Now I wanted to undo my changes. Now I realise that I can't. When I click on the X, the data is saved immediately.

A save and / or cancel button would be nice.

**Comment by feraswilson** [comment]
Yes, the data is actually saved directly. It is by design, which I thought was easier because it reduces the steps needed to make a workflow. I understand your issue and maybe a solution for this is revisions, the ability to go back.
